docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,63 @@
+- [.NET Core samples](https://github.com/dotnet/core)
+- [.NET Core OS and CPU Support](link)
+- [Explore .NET Core repos](https://github.com/dotnet/core)
+
+Composition of .NET Core
+------------------------
+
+**.NET Core is the following:**
+
+- **A set of packages** -- These contain the runtime and the API shape and implementation of .NET Core.
+- **A set of metapackages (e.g. .NET Standard Library)** --  They describe various layers of .NET Core by referencing the appropriate set of library packages.
+- **A [distribution](https://aka.ms/dotnetcoregs) of .NET Core from Microsoft** -- This includes the CoreCLR runtime, associated libraries, the console application host and the `dotnet` launcher. It is described by the Microsoft.NETCore.App metapackage.
+- **A [distribution](https://dotnet.github.io) of the .NET Core SDK from Microsoft** -- This includes a set of tools for restoring NuGet packages and compiling and building apps, also controlled from the `dotnet` launcher. It is extensible, enabling anyone to add commands. The package also includes a copy of .NET Core to use with the tools. Typically, developers will install this distribution.
+
+**Closely affiliated with .NET Core:**
+
+- **The ".NET Standard Library"** - This is the API spec that describes the API evolution of .NET and that .NET Core implements. While many of the additions to the Standard Library will come from API additions to .NET Core, some will come from other platforms, such as the .NET Framework and Mono.
+- **System.* packages** - These packages provide the ".NET Core" implementation but contain assets for other platforms, such as the .NET Framework and Mono.
+
+Designed for Adaptibility
+-------------------------
+
+.NET Core has been built as a very similar but unique product relative to other .NET products from Microsoft. The primary reason to make changes was to enable broad adaptibility, to new platforms, for new workloads and by new compiler toolchains. In particular, apps should only need to use and distribute the .NET Core libraries and subsystems that they actually depend on. This characteristic is particularly important where either storage space and/or wire cost (download time, data charges) are important. It should also be the case that they do not have to change their code to target a new workload of compiler toolchain. The architecture of .NET Core should enable those outcomes.
+
+By comparison, The .NET Framework was built as a completely integrated product, providing end-to-end experiences across Windows, Visual Studio and even SQL Server. Most customers loved that level of integration, but it was difficult to produce meaningful subsets of it. For example, because of API factoring, some subsystems are always required (e.g. Object.GetType has a dependency on Reflection). With .NET Core, those hard API dependencies have been removed and the APIs redefined, to enable the goal of adaptibility.
```

Comment:
Not sure why I'd provide links. I'm not trying to get people to learn more about them. I kinda get it, but it doesn't seem super useful.